### PR TITLE
Vv 139 correlation

### PR DESCRIPTION
**Pull request by Richard-Pentecost**
# What

# Why

# Screenshots

# Notes

### CODE REVIEW

**Comment by nax2uk** [comment]
Hi Richard, can you resolve the conflicts please?

File: src/helpers/apiErrorHandler.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ export function validateHttpMethod(validMethod, method, res) {
 }
 
 export function checkIfAuthorised(token, res) {
+  console.log("CheckIfAuthorised")
```
[the remark this answers is not in the document]
Comment (Richard-Pentecost) [reply src/helpers/apiErrorHandler.js]: Thanks Azlina, should be done now

**Review by nax2uk — COMMENTED** [review]
I think these are the old api and test files which you have moved to pages/api/test-endpoint/test-send-reset-password-email.js in master?:

pages/api/test-send-email.js
test/pages/api/test-send-email.test.js

If so please delete them. Thanks.

**Comment by Richard-Pentecost** [comment]
Just deleted those 2 files.

**Review by nax2uk — APPROVED** [review]

**Pull request merged**